__test__/url-helper.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as urlHelper from '../src/url-helper'
+
+describe('isGhes tests', () => {
+  it('basics', async () => {
+    expect(urlHelper.isGhes()).toBeFalsy()
+    expect(urlHelper.isGhes('https://github.com')).toBeFalsy()
+    //expect(urlHelper.isGhes('https://api.github.com')).toBeFalsy()
```

Comment:
This assertion currently fails when uncommented.

Reply:
You can remove this test case as the URL should always be the base URL and not include a subdomain of `github.com`.